variables.tf

```diff
@@ -215,6 +215,54 @@ EOF
 # TLS
 # --------------------
 
+variable user_managed_tls {
+  type = "string"
+  default = false
+  description = <<EOF
+Set to true if you'd like to manage and upload your own TLS files, if you do not want this module
```

Comment:
To ensure BC, I think this should be `manage_tls` and it should default to `true`.